test.py

```diff
@@ -15,27 +56,33 @@ def test_skips_noqa(self):
         result = check_code_for_print_statements('print(4) # noqa')
         assert_equal(result, list())
 
+    def test_skips_noqa_multiline(self):
+        s = dedent("""
+            print("a"
+                  "b")  # noqa
+        """).strip()
```

Comment:
I'll take a look at this tonight. Sorry it has taken so long, I was at
Pycon and had to prepare the last few weeks. But now that the conference is
over, I'll reply tonight.

On Mon, Nov 9, 2015, 7:10 PM John Vandenberg <notifications@github.com>
wrote:

> In test.py
> <https://github.com/JBKahn/flake8-print/pull/12#discussion_r44353570>:
>
> > @@ -15,27 +56,33 @@ def test_skips_noqa(self):
> >          result = check_code_for_print_statements('print(4) # noqa')
> >          assert_equal(result, list())
> >
> > +    def test_skips_noqa_multiline(self):
> > +        s = dedent("""
> > +            print("a"
> > +                  "b")  # noqa
> > +        """).strip()
>
> this strip() isnt needed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JBKahn/flake8-print/pull/12/files#r44353570>.
>